index.js

```diff
@@ -2,7 +2,110 @@ const menuBox = document.getElementById('menuBox');
+  },
+];
+
+// SHOWING DATA OF ARRAY FOR CARDS dynamically
+arr.forEach((item, index) => {
+  htr.innerHTML += `<article class="cards">
+ <div class="${cardDirection[index]}">
+     <a href="#" class="mobile-pic" ><img class="project-pics" src="${item.image}" alt="tonic Project image"></a>
+     <a href="#" class="desktop-pic" ><img class="project-pics" src="${item.image}" alt="tonic Project image"></a>
+    <div class="flow">
+     <h3 class="project-title">${item.name}</h3>
+     <ul class="info">
+         <li class="conopy-li">
+             <h3 class="canopy">${item.client[0]}</h3>
+         </li>
+         <li class="couter-image">
+                 <img src="./images/Counter.png" alt=" ">
+         </li>
+         <li class="back-end-li">
+             <h3 class="back-end">${item.client[1]}</h3>
+         </li>
+         <li class="couter-image">
+                 <img src="./images/Counter.png" alt=" ">
+         </li>
+         <li>
+             <h3 class="year">${item.client[2]}</h3>
+         </li>
+     </ul> 
+ <p class="info-about">${item.dicsription}</p>
+     <ul class="info-2">
+         <li>
+             <button type="submit" class="button-1">${item.technologies[0]}</button>
+                 
+         </li>
+         <li>
+             <button type="submit" class="button-2">${item.technologies[1]}</button>
+                 
+             
+         </li>
+         <li>
+             <button type="submit" class="button-3">${item.technologies[2]}</button>
+                 
+             
+         </li>
+     </ul>
+         <button type="submit" class="button-4">See Project</button>
+   
+ </div>
+ 
+ </div>
+ </article>`;
+});
+
+// SHOWING DATA OF ARRAY FOR Popupmenu dynamically
+document.querySelectorAll('.button-4').forEach((n, index) => {
+  n.addEventListener('click', () => {
+    title.innerText = arr[index].name;
+    dis.innerText = arr[index].dicsription;
+    imge.src = arr[index].image;
+    livelink.onclick = arr[index].live_version;
+    githublink.onclick = arr[index].source;
+    overlay.style.display = 'block';
+  });
+});
```

Comment:
- [ ] Excellent job making changes requested by the previous reviewer 💯, but there is a little detail, please as @codecaiine says [here](https://github.com/ahadb802/My-Portfolio/pull/13#discussion_r1024612785), you should stick as much as possible with Figma design. Kindly, the title, image, and description of the project in the popup window should be aligned. 🎉 

Your design:

![image](https://user-images.githubusercontent.com/91301423/202328518-be73aeb5-3f06-4442-b135-059da6b0141b.png)

Figma design: 

![image](https://user-images.githubusercontent.com/91301423/202328711-acba703b-3ad2-4f28-abfa-fba478189982.png)